src/OpenLoco/OpenLoco.cpp

```diff
@@ -229,11 +229,6 @@ namespace OpenLoco
         }
     }
 
-    uint32_t scenarioTicks()
-    {
-        return _scenario_ticks;
```

Comment:
Can the loco_global in question be deleted, too?

Reply:
it can, it's missing here because I messed up a rebase. fixed now